### ISSUE TITLE
[nsfw] hotpornfile.org: breakage

### ISSUE THREAD

**Issue by fl4shforward** [uBlockOrigin/uAssets#21367]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.hotpornfile.org/dirtyauditions-22-12-29-khloe-kapri-xxx-1080p-mp4-nbq/783003`

### Category

breakage

### Description

When clicking on download and get links, a js alert shows:

> Your browser is blocking the loading of a new tab.

And downloads links never show up.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.54.0
Chromium: 120
filterset (summary):
 network: 134980
 cosmetic: 203622
 scriptlet: 27227
 html: 0
listset (total-discarded, last-updated):
 added:
  https://filters.adtidy.org/extension/ublock/filters/14.txt: 76077-120, now
  https://secure.fanboy.co.nz/fanboy-annoyance_ubo.txt: 95026-21932, now
  adguard-social: 22084-1620, now
  fanboy-cookiemonster: 49114-49113, now
  fanboy-social: 15986-15986, now
  fanboy-thirdparty_social: 69-18, now
 default:
  user-filters: 9-0, never
  FRA-0: 28803-76, now
  easylist: 75532-158, now
  easyprivacy: 33129-536, now
  plowe-0: 3777-1159, now
  ublock-badware: 7606-153, now
  ublock-filters: 36004-614, now
  ublock-privacy: 937-15, now
  ublock-quick-fixes: 91-17, now
  ublock-unbreak: 2163-38, now
  urlhaus-1: 11190-0, now
filterset (user): [array of 9 redacted]
trustedset:
 added: [array of 96 redacted]
 removed:
  edge-scheme
switchRuleset:
 added: [array of 2 redacted]
hostRuleset:
 added: [array of 2 redacted]
userSettings:
 advancedUserEnabled: true
 cloudStorageEnabled: true
 showIconBadge: false
hiddenSettings: [none]
supportStats:
 allReadyAfter: 251 ms (selfie)
 maxAssetCacheWait: 187 ms
popupPanel:
 blocked: 11
 network:
  hotpornfile.org: 4
  alleviatepracticableaddicted.com: 1
  googletagmanager.com: 1
  purdasseer.com: 4
  sadjklq.com: 1
 extended:
  ##.grecaptcha-badge
  ##a[href*="theporndude.com"]
  ##+js(abort-on-property-write, getIfc)
  ##+js(addEventListener-defuser, getexoloader)
  ##+js(no-setTimeout-if, _0x)
  ##+js(no-window-open-if, !bergblock)
  ##+js(set-constant, adblockcheck, false)
  ##+js(set-constant, rwt, noopFunc)
```
</details>

**Comment by huyszn** [comment]
@ghajini Scroll down until you see the "Download" button. Click on the button and then click on the "Get Links" button and you will see the alert. **NSFW** https://github.com/uBlockOrigin/uAssets/assets/46979193/e328e82a-8ab4-4544-b0f9-9c6fe8caa8dc

**Comment by huyszn** [comment]
I cleared cookies from the site and was able to see the alert again after unlocking the download links with uBO disabled.

**Comment by mapx-** [comment]
I guess they are checking the cookies (set when you click the first ad on that search).
However, to unlock all this crap we should add lots of exceptions or - maybe - create lots of cookies.
Who wants to use download should whitelist hotpornfile only during the download.

**Comment by okiehsch** [comment]
It's caused by `hotpornfile.org##+js(nowoif, !bergblock)` on my end.
Hm. I wonder if other nowoif filters will lead to the same 
"Your browser is blocking the loading of a new tab."
message, that could be a problem.

**Comment by okiehsch** [comment]
I tested some sites with working nowoif filters and no issue theres.
I wonder why it happens on this site.

**Comment by mapx-** [comment]
it seems these could work (but you have to repeat at least 3 times the clicking / waiting):

```
hotpornfile.org#@#+js(no-window-open-if, !bergblock)
hotpornfile.org##+js(no-window-open-if, !bergblock, 10)
@@||searchwithme.net/redirect?tid=$subdocument,domain=hotpornfile.org
```

**Comment by okiehsch** [comment]
>  I repeated the test and now on the first try it showed the links

That's because the site sets a cookie.

I never saw that kind of message in a browser caused by an extension.
I get the same message with chrome and firefox.

**Comment by mapx-** [comment]
I deleted all cookies before testing.
Did the filters above work for you ?

We have other 17 filters needing that third parameter:

search uBO filters using
`/nowoif, .*\,/`

**Comment by okiehsch** [comment]
> Did the filters above work for you ?

Yes.

**Comment by okiehsch** [comment]
> We have other 17 filters needing that third parameter:

Yes, but but I tried removing that parameter on one site and I still can't reproduce the
`Your browser is blocking the loading of a new tab."
message, so I don't know what exactly causes it, it can't only be our filter.

**Comment by mapx-** [comment]
It's not a browser thing, it's simply an alert message programmatically inserted by the page

**Comment by mapx-** [comment]
Maybe the third param was added when we first meet this trick (without alert; the page is checking if the object window is still alive)  for those arab video pages (I dont remember the name)
